src/main/java/net/sf/jabref/gui/prrv/ReferenceRelationship.java

```diff
@@ -0,0 +1,136 @@
+package net.sf.jabref.gui.prrv;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.sf.jabref.JabRefGUI;
+import net.sf.jabref.gui.FXDialogs;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.model.entry.BibEntry;
+
+import prefux.data.Edge;
+import prefux.data.Graph;
+
+/**
+ * @author Daniel Brühl
+ */
+public class ReferenceRelationship {
+
+    // List to store the temporary data for graph visualisation
+    private List<BibEntry> pureEntryList = JabRefGUI.getMainFrame().getCurrentBasePanel().getBibDatabaseContext().getDatabase().getEntries();
+    public List<prefux.data.Node> entryNodeList = new ArrayList<>();
+    public List<prefux.data.Node> externalNodeList = new ArrayList<>();
```

Comment:
Does this need to be public? If so please make them private and use getter/setter methods.